### ISSUE TITLE
Кейтеринг 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#279]
Всем привет. У нас в офисе намечается крупная мероприятие, посоветуйте мне выездной кейтеринг где есть в Челябинске действительно эстетический красивый и привлекательный, вкусный

**Comment by diorjio** [comment]
Конечно, если говорить о качественном кейтеринге в Челябинске, я однозначно рекомендую компанию "Сели Съели" https://sitandeat.ru/keytering/ . Мы недавно отмечали юбилей моей мамы, и я хотела, чтобы все было идеально, особенно еда. Заказали у них фуршетное меню, и я была просто в восторге. У них действительно широкий выбор блюд на любой вкус: мини-десерты, выпечка, мини-закуски, фруктовые плато, различные рулетики и волованы. Еда была не только красивой, но и очень вкусной. Всё приготовлено профессионально, блюда приехали горячими и свежими, что не всегда бывает с кейтерингом. Кроме того, у них отличные условия: возможность собрать индивидуальное меню, быстрая доставка и даже акции и скидки на некоторые предложения. Если вам нужно срочно, они могут приготовить буквально за 3 часа. Рекомендую их всем, кто хочет организовать мероприятие на высшем уровне.
